test/proposals/Proposal_0.sol

```diff
@@ -142,13 +145,16 @@ contract Proposal_0 is Proposal {
                     hardCap: 2_000_000e18 // max 2M CREDIT issued
                 })
             );
-            LendingTerm termSDAI1 = new LendingTerm(
+            LendingTerm termSDAI1 = new LendingTerm();
```

Comment:
LendingTerm should be created through the factory, not by calling the contract itself.